docs/content/app-protect-dos/installation.md

```diff
@@ -20,7 +20,7 @@ This document provides an overview of the steps required to use NGINX App Protec
 2. Clone the Ingress Controller repo:
 
     ```
```

Comment:
```suggestion
    ```shell
```
Code blocks should to have a language defined (even if it's `none`). For shell instructions use `shell` or `bash` (please note that `console` is not recognized by hugo and breaks the highlighting)

Reply:
we might need to open a ticket to review all the code blocks in the docs and add the missing languages cc @ADubhlaoich